event-grid/filter-events/filter-events.ps1

```diff
@@ -0,0 +1,15 @@
+﻿# Provide an endpoint for handling the events.
+$myEndpoint = "<endpoint URL>"
+
+# Select the Azure subscription that contains the resource group.
+Set-AzureRmContext -Subscription "Contoso Subscription"
+
+# Get the resource ID to filter events
+$resourceId = (Get-AzureRmResource -ResourceName demoSecurityGroup -ResourceGroupName myResourceGroup).ResourceId
```

Comment:
Resource name and group should be user-supplied values. Any objects which are needed in the script should be created as part of running it.